factoring/interfaces.py

```diff
@@ -74,7 +92,7 @@ def factor(P, use_saved_embedding=True):
     sample_time = time.time()
 
     # get QPU sampler
-    sampler = DWaveSampler()
+    sampler = DWaveSampler(solver_features=dict(online=True, name='DW_2000Q.*'))
```

Comment:
Note: in prod, when list of solver is limited for each user, `name` can be left out -- `online=True` is enough.